proto/minder/v1/minder.proto

```diff
@@ -854,6 +854,8 @@ message GetAuthorizationURLRequest {
     // config is a JSON object that can be used to pass additional configuration
     google.protobuf.Struct config = 8;
 
+    string provider_class = 9;
```

Comment:
the session store was only using the provider name, so I added class as a hit on what's being enrolled.